WIC.py

```diff
@@ -0,0 +1,72 @@
+import re
+import pdfplumber
+import pandas as pd
+
+
+'''
+GCFD staff provided a pdf IWIC.pdf which contains WIC data for 96 out of 102 IL Counties. We do not know why some counties
+are missing. Data is for the month of January year 2021. WIC.py reads thru each line of each page of the pdf looking
+ for lines that contain the data we want and then saves that data to a dataframe. GCFD staff also requested a .csv version
+ of the data'''
+
+# use regular expression matching to find lines that start with certain words and save as variables:
+# find rows that start with Total (this includes Total Women, Total Infant and Total Children rows)
+Total_re = re.compile("Total")
+# find rows that start with LA Total (these rows include the subtotal values for the specific County)
+LA_Total_re = re.compile("LA Total")
+# find rows that start with three digits (these rows contain County ID and name)
+County_re = re.compile(r"\d\d\d")
+
+# define column names
+# readable column names
+column_names = ["County_ID", "County", "WIC1", "WIC2", "Amer. Indian or Alaska Native",
+                "Asian", "Black or African American", "Native Hawaii or Other Pacific Isl.",
+                "White", "Multi Racial", "Total_Participants", "Hispanic_Latino"]
+
+# create empty data frame with column names
+data = pd.DataFrame(columns=column_names)
+
+# read each page of the pdf pages 1 thru 97. Skip page 0 because it shows Statewide totals which we don't need
+pages = list(range(1, 97))
+
+# filepath to the PDF we want to read data from
+filepath = "C:/Users/User1/PycharmProjects/greater-chicago-food-despository/data_folder/IWIC.pdf"
+
+# iterate thru pages 1 to 97
+for p in pages:
+    # read each page using pdfplumber
+    with pdfplumber.open(filepath) as pdf:

```

Comment:
Does it take a more than a few seconds to run this script? It looks like we're reopening the pdf to just read one page. Opening files can be relatively slow, so if we need to speed up the script I think we could switch the lines "for p in pages:" and "with pdfplumber.open(filepath) as pdf:"